src/Event/NotificationEvent.php

```diff
@@ -13,22 +13,37 @@
 
 class NotificationEvent extends ThemeEvent
 {
-    /**
-     * @var array<NotificationInterface>
-     */
+    public ?string $title = null;
```

Comment:
no public variables. only setter and getter

Reply:
MB !!! 💀